.github/workflows/tests.yml

```diff
@@ -27,23 +27,30 @@ jobs:
     - name: Install Ruby gems with Bundler
       run: bundle install
     - name: Lint podspec using local source
-      run: pod lib lint --verbose ${{ matrix.flags }}
+      run: pod lib lint --verbose ${{ matrix.flags }} --allow-warnings
 
   spm-build-test:
     runs-on: macOS-latest
+    strategy:
+      matrix:
+        include:
```

Comment:
Although this works, for clarity, I think we should add an entry above defining the matrix into which we want to 'include' values:

sdk: ['macos', 'iphonesimulator']